local-cli/generator-macos/templates/macos/Podfile

```diff
@@ -25,24 +68,45 @@ abstract_target 'Shared' do
   pod 'React-jsi', :path => '../node_modules/react-native-macos/ReactCommon/jsi'
   pod 'React-jsiexecutor', :path => '../node_modules/react-native-macos/ReactCommon/jsiexecutor'
   pod 'React-jsinspector', :path => '../node_modules/react-native-macos/ReactCommon/jsinspector'
-  pod 'ReactCommon/jscallinvoker', :path => '../node_modules/react-native-macos/ReactCommon'
+  pod 'ReactCommon/callinvoker', :path => '../node_modules/react-native-macos/ReactCommon'
   pod 'ReactCommon/turbomodule/core', :path => '../node_modules/react-native-macos/ReactCommon'
-  pod 'Yoga', :path => '../node_modules/react-native-macos/ReactCommon/yoga'
+  pod 'Yoga', :path => '../node_modules/react-native-macos/ReactCommon/yoga', :modular_headers => true
 
-  pod 'boost-for-react-native', :podspec => '../node_modules/react-native-macos/third-party-podspecs/boost-for-react-native.podspec'
   pod 'DoubleConversion', :podspec => '../node_modules/react-native-macos/third-party-podspecs/DoubleConversion.podspec'
   pod 'glog', :podspec => '../node_modules/react-native-macos/third-party-podspecs/glog.podspec'
   pod 'RCT-Folly', :podspec => '../node_modules/react-native-macos/third-party-podspecs/RCT-Folly.podspec'
+  pod 'boost-for-react-native', :podspec => '../node_modules/react-native-macos/third-party-podspecs/boost-for-react-native.podspec'
 
   target 'HelloWorld-macOS' do
     platform :macos, '10.14'
     use_native_modules!
+
+    # Enables Hermes
+    #
+    # Be sure to first install the `hermes-engine-darwin` npm package, e.g.:
+    # 
+    #   $ yarn add --dev 'hermes-engine-darwin@^0.4.3'
+    #
+    # pod 'React-Core/Hermes', :path => '../node_modules/react-native-macos/'
+    # pod 'hermes', :path => '../node_modules/hermes-engine-darwin'
+    # pod 'libevent', :podspec => '../node_modules/react-native-macos/third-party-podspecs/libevent.podspec'
```

Comment:
After enabling this:

<img width="812" alt="Screenshot 2020-08-18 at 16 55 06" src="https://user-images.githubusercontent.com/2320/90547606-160f6480-e18c-11ea-9a98-edbbaa755800.png">